### ISSUE TITLE
when displaying help information on elasticsearch deprecation log is misleading about its location

### ISSUE THREAD

**Issue by majormoses** [elastic/elasticsearch-migration#117]
This makes the assumption that the cluster name has not been changed from the default of `elasticsearch`. I had to spend some time convincing a developer and had to go to the source to confirm this is no indication of the actual filename and is just the default location.

https://github.com/elastic/elasticsearch-migration/blob/3f3f2d4f71050a3a4d89fe681a2c63180081bab6/_site/index.html#L148-L151

I think we should either enhance the verbiage to say something like this:
```
Deprecation logs can be found in the logs directory, in the file named `$CLUSTERNAME_deprecation.log`
```

Or even better would be to actually just return the value from the es query as you already have this info anyways. For example:
```
$ curl -s -S $REDACTED:9200/_cluster/state | jq .cluster_name
"es-app"
$ curl -s -S $REDACTED:9200 | jq .cluster_name
"es-app"
```
This could be javascriptified I am sure but I just wanted to see what peoples thoughts were on this before spending time hacking javascript which I don't really touch front end code so I am sure it would horrify most.

**Comment by clintongormley** [comment]
Good point @majormoses.  That said, this migration tool is unlikely to see another release now, so it is probably not worth fixing at this point.
